generate_numpy_array.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
+from numpy import array
+
+
+def generate_numpy_array() -> array:
+    """
+    The function generates a numpy array with values from 0-20.
+    Then it changes the sign of the numbers in the range from 0 to 15.
+    :return: A numpy array
+    """
+    return np.concatenate((np.arange(9), (np.arange(7)+9)*-1, (np.arange(5)+16)))
```

Comment:
I recommend using concatenate only when you really have to. As you learnt numpy array sizes are determined upon creation (and so they are contiguous in memory), so changing the sizes of numpy arrays is not an efficient approach (basically it deletes the numpy array and creates a new one).